packages/shared/lib/migration.ts

```diff
@@ -143,44 +157,52 @@ export const createMigrationBundle = (inputAddressIndexes: number[], offset: num
  * @returns {Promise<void>}
  */
 export const sendMigrationBundle = (bundleHash: string, mwm = MINIMUM_WEIGHT_MAGNITUDE): Promise<void> => {
-    return new Promise((resolve, reject) => {
-        api.sendMigrationBundle([MIGRATION_NODE], bundleHash, mwm, {
-            onSuccess(response) {
-                const { bundles } = get(migration);
-
-                // Update bundle and mark it as migrated
-                bundles.update((_bundles) => {
-                    return _bundles.map((bundle) => {
-                        if (bundle.bundleHash === bundleHash) {
-                            return Object.assign({}, bundle, { migrated: true })
-                        }
-
-                        return bundle
+    return new Promise(async (resolve, reject) => {
+        // Migration: snapshot check
+        await checkChrysalisSnapshot()
+        if (get(ongoingSnapshot) === true) {
+            // we dont display the error because a notification popup will open already
```

Comment:
How come? Say the network stopped while the user was migrating multiple bundles?

Reply:
We won't display an error notification, a red toast, but we will open the popup for the user and the bundle transaction will still display an error, maybe the message is confusing